package.json

```
@@ -85,8 +85,11 @@
     "webdriverio": "^8.23.1"
   },
   "dependencies": {
-    "expect": "^29.7.0",
+    "@jest/expect": "^29.7.0",
     "jest-matcher-utils": "^29.7.0",
     "lodash.isequal": "^4.5.0"
+  },
+  "peerDependencies": {
+    "jest-snapshot": "29.7.0"
```

Since `@jest/expect` has this dependency defined with the same `^`
```suggestion
    "jest-snapshot": "^29.7.0"
```